resources/system-overview.md

```diff
@@ -0,0 +1,105 @@
+# The Shape of Eligibility Systems
+
+It can be helpful to undertand the general shape of the systems that you will be intereacting with. There is much variation in capability, acrhtiecture, and even nomenclature across states and vendors. This document tried to capture some generalities.
+
+## System Overview
+At the widest perspective the Medicaid eligibility system includes a number of interconnected components: a datastore, a system implementing the business logic of the program, often a separate “rules engine” that implements the business logic specific to eligibility determinations, connections to a variety of state and federal data sources, a system for generating notifications across a range of media, a number of specialized user interfaces for different audiences, and a mechanism for generating reports and dashboards.
+
+These systems process and store both PII and PHI, which adds additional constraints to their design.
+
+![block diagram of a medicaid elgibility system including web portals, buisness logic, rules engines, and data sources](/images/system_diagram.svg)
```

Comment:
We've been setting images using HTML, with a block like this:

```html
<div align="center">
  <img
    width="500px"
    src="/images/system_diagram.svg"
    alt="block diagram of a medicaid elgibility system including web portals, buisness logic, rules engines, and data sources" />
</div>
```

It keeps the image centered and ensures it doesn't blow up to unreasonable sizes on some screens.

Reply:
done, much nicer, thanks